.github/workflows/ci.yml

```diff
@@ -0,0 +1,171 @@
+  CI_ENABLED: true
+
+jobs:
+# TODO: Fix Markdown-link-check failures https://github.com/astronomer/astronomer-cosmos/issues/172
+#  Markdown-link-check:
+#    if: github.event.action != 'labeled'
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v3
+#      - uses: gaurav-nelson/github-action-markdown-link-check@v1
+#        with:
+#          config-file: '.github/workflows/mlc_config.json'
+#
+# TODO: Fix Type-check failures https://github.com/astronomer/astronomer-cosmos/issues/173
+#  Type-Check:
+#    if: github.event.action != 'labeled'
+#    runs-on: ubuntu-latest
+#    env:
+#      MYPY_FORCE_COLOR: 1
+#      TERM: xterm-color
+#      SETUPTOOLS_USE_DISTUTILS: stdlib
+#    steps:
+#      - uses: actions/checkout@v3
+#      - uses: actions/setup-python@v3
+#        with:
+#          python-version: '3.8'
+#          architecture: 'x64'
+#      - uses: actions/cache@v3
+#        with:
+#          path: |
+#            ~/.cache/pip
+#            .nox
+#          key: ${{ runner.os }}-${{ hashFiles('pyproject.toml') }}
+#      - run: pip3 install nox
+#      - run: nox -s type_check
+
+  Build-Docs:
+    if: github.event.action != 'labeled'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.9'
+          architecture: 'x64'
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/pip
+            .nox
+          key: ${{ runner.os }}-${{ hashFiles('pyproject.toml') }}
+      - run: pip3 install nox
+      - run: nox -s build_docs
+
+  Run-Unit-tests-Airflow-2-5:
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [ '3.8', '3.9', '3.10' ]
+        airflow: [ 2.5 ]
+    if: >-
+      github.event_name == 'push' ||
+      (
+        github.event_name == 'pull_request' &&
+        github.event.pull_request.head.repo.fork == false
+      ) ||
+      (
+        github.event_name == 'pull_request_target' &&
+        contains(github.event.pull_request.labels.*.name, 'safe to test')
+      ) ||
+      (
+        github.event_name == 'release'
+      )
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        if: github.event_name != 'pull_request_target'
+
+      - name: Checkout pull/${{ github.event.number }}
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+        if: github.event_name == 'pull_request_target'
+      - uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.version }}
+          architecture: 'x64'
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/pip
+            .nox
+          key: tests-os-${{ runner.os }}-python-${{ matrix.python }}-airflow-${{ matrix.airflow }}-deps-${{ hashFiles('pyproject.toml') }}-version-${{ hashFiles('cosmos/__init__.py') }}
```

Comment:
@pankajkoti and I checked which version customer B is using, and they are using Airflow 2.2.4.

We agreed to automate the checks (tests & example DAGs) for this version of Airflow as well in a separate ticket:
https://github.com/astronomer/astronomer-cosmos/issues/174